bundle/compliance/policy/azure/data_adapter.rego

```diff
@@ -17,3 +17,7 @@ is_storage_account {
 is_storage_account {
 	input.type == "azure-classic-storage-account"
 }
+
+is_website_asset {
+	input.type == "azure-web-site"
```

Comment:
still no idea why you split it into two words lol

Reply:
yeah just went with it. eventually we'll generate the `type/subtype` from the asset type's string. no need to invent new strings